website/client/src/components/userMenu/profileStats.vue

```diff
@@ -684,4 +693,37 @@ export default {
   .save-row {
     margin-top: 1em;
   }
+
+
+    .gear.box {
+      vertical-align: top;
+      margin: 0 auto;
+    }
+
+    .gear-label {
+      margin: 0 auto;
+      margin-top: 0.5rem;
+      min-height: 1rem;
+      font-family: Roboto;
```

Comment:
is this rule alongside `font-stretch: normal;` `font-style: normal;` and `letter-spacing: normal;` necessary?

Reply:
gonna double check if it can used without or if there is a higher class/rule that overrides the "inherit" value

Reply:
removed them :)